### PR TITLE
Andrey amazon s3

### PR DESCRIPTION
**Pull request by AndreyStadnyk**
opportunity of uploading userPic was added

### CODE REVIEW

File: src/main/java/ua/com/danit/controller/FileController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import ua.com.danit.dto.request.UserRequest;
+import ua.com.danit.dto.response.GenericResponse;
+import ua.com.danit.service.FileService;
+
+@RestController
+@RequestMapping("/api")
```
Comment (alex5272372) [inline src/main/java/ua/com/danit/controller/FileController.java]: Change to /api/img

File: src/main/java/ua/com/danit/controller/FileController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+public class FileController {
+  private FileService fileService;
+
+  @Autowired
+  public FileController(FileService fileService) {
+    this.fileService = fileService;
+  }
+
+  @RequestMapping(value = "/upload/userPic", method = RequestMethod.POST, consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```
Comment (alex5272372) [inline src/main/java/ua/com/danit/controller/FileController.java]: Change to /user-pic

File: src/main/java/ua/com/danit/controller/FileController.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  public GenericResponse uploadFile(@RequestParam("file") MultipartFile file) {
+    return fileService.uploadUserPic(file);
+  }
+
+  @GetMapping("/getUserPicPathByUsername")
```
Comment (alex5272372) [inline src/main/java/ua/com/danit/controller/FileController.java]: Change to /user-pic

File: src/main/java/ua/com/danit/controller/FileController.java (hunk 1 above)
Comment (AndreyStadnyk) [reply src/main/java/ua/com/danit/controller/FileController.java]: +

File: src/main/java/ua/com/danit/controller/FileController.java (hunk 2 above)
Comment (AndreyStadnyk) [reply src/main/java/ua/com/danit/controller/FileController.java]: +

**Review by ShevchukMarina — APPROVED** [review]

**Review by skosinskiy — APPROVED** [review]

File: src/main/java/ua/com/danit/controller/FileController.java (hunk 3 above)
Comment (AndreyStadnyk) [reply src/main/java/ua/com/danit/controller/FileController.java]: +

**Pull request merged**